packages/appeals-service-api/src/db/seed/data-dev.js

```diff
@@ -386,6 +387,14 @@ async function seedDev(dbClient) {
 		});
 	}
 
+	for (const appeal of lpaAppeals) {
+		await dbClient.appeal.upsert({
+			create: appeal,
+			update: appeal,
+			where: { id: appeal.id }
+		});
+	}
+
```

Comment:
could we just spread these into the existing arrays instead of a separate for + upsert?

Reply:
Yep, good point!